src/PhpSpreadsheet/Calculation/Web.php

```diff
@@ -18,7 +19,7 @@ class Web
      *
      * @return string the output resulting from a call to the webservice
      */
-    public static function WEBSERVICE(string $url)
+    public static function WEBSERVICE(string $url, ?ClientInterface $client = null)
```

Comment:
The method signature must not change, because in most cases the user is not directly calling this method, instead the calculation engine is. And the calculation engine cannot provide extra parameters such as this new one.